ko/sql/query/select.rst

```diff
@@ -159,6 +159,7 @@ FROM 절
         <metaclass_specification> [<correlation>] |
         <subquery> <correlation> |
         TABLE (<expression>) <correlation>
```

Comment:
It seems that '|' should be added at the end.

```
TABLE (<expression>) <correlation>
-> TABLE (<expression>) <correlation> |
```

Reply:
I see, Thank you!